content/20.treatments.md

```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+In a recent review [@doi:10.1038/s41577-020-0285-6], trained immunity was defined as forms of memory which are temporary (e.g., months or years, and reversible), displayed by innate immune cells and innate immune features of other cells, displayed as increased responsiveness to future same or heterologous pathogen infection or sometimes decreased responsiveness or immunological tolerance, and established through epigenetic and metabolic mechanisms; the review also notes various ways in which trained immunity may be related to pathology, inflammatory-related diseases, conditions and immunology associated with aging, and cancer.
+One type of stimulus which research indicates can induce trained immunity is BCG vaccination (BCG is an acronym for bacillus Calmette-Guerin).
+BCG is a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
+Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
+One set of trials for example has time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
+An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many other possible factors or confounding factors (country age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, many other mundane factors) and (2) also it is unclear what is the relation between BCG vaccination at different ages and at longer times before the start of the SARS-CoV-2 epidemic and BCG vaccination at different ages during the SARS-CoV-2 epidemic at shorter times before the risk of possible infection; at least some of these considerations are also made or pointed out in the data analyses sited and in the article.
```

Comment:
i don't think "mundane" is typical outside of philosophizing english teachers or other esoteric examples ;-)

Reply:
- your re-wording of 2nd point covers everything / is more general, though I did mean something specific; I think your more wording may be better than mine because it covers several possibilities.
- i don't think "mundane" is a typical word! I sort of meant that I listed obviously possibly related factors but sometimes you also find factors that aren't obviously related, but I don't have enough experience to have an intuition or estimate of how often that is. So we can either just say "and other factors" removing the "mundane" or we can remove last comma clause and put "and" before "differences".

Reply:
That sounds good to me!